src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/ItemTooltip.java

```diff
@@ -398,11 +398,15 @@ private static void smoothenLines(List<Text> lines) {
         for (int i = 0; i < lines.size(); i++) {
             Text line = lines.get(i);
             if (line.getString().equals("-----------------")) {
-                lines.set(i, Text.literal("                    ").formatted(Formatting.DARK_GRAY, Formatting.STRIKETHROUGH, Formatting.BOLD));
+                lines.set(i, createSmoothLine());
             }
         }
     }
 
+    public static Text createSmoothLine() {
+        return Text.literal("                    ").formatted(Formatting.DARK_GRAY, Formatting.STRIKETHROUGH, Formatting.BOLD);
```

Comment:
It would be better to detect the colour of the line rather than assume it which could break things

Reply:
Probably just apply the thing to lines of the same colour to avoid any unexpected changes